apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -335,6 +344,103 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
     }
   }
 
+  const getMetadataLabels = async () => {
+    let chatMetadata: any
+    try {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)